streamlit_echarts/frontend/src/EchartsChart.tsx

```diff
@@ -73,12 +74,33 @@ const EchartsChart = (props: ComponentProps) => {
   const cleanTheme = registerTheme(theme)
 
   if (isObject(map)) {
-    echarts.registerMap(map.mapName, map.geoJson, map.specialAreas)
+      echarts.registerMap(map.mapName, map.geoJson, map.specialAreas)
+
   }
 
   const cleanOptions = convertJavascriptCode(options)
   const cleanOnEvents = convertJavascriptCode(onEvents)
 
+  /*
+  add support for event back to python
+  usage at python side:
+  events={
+    "click": "function(params) { return params.name }",
+    "dblclick":"function(params) { return params.value }"
+  }
+  s=st_pyecharts(b,events=events)
+   */
+  const keys=Object.keys(cleanOnEvents)
+  const getReturnOfcleanOnEvents:any={}
+  keys.forEach(
```

Comment:
you can write `Object.keys(cleanOnEvents).forEach(function (key) {` directly :) I think it's clear enough